packages/mobile/test/schemas.ts

```diff
@@ -930,6 +930,7 @@ export const v31Schema = {
     biometryEnabled: false,
     supportedBiometryType: null,
     useBiometry: false,
+    linkBankAccountStepTwoEnabled: false,
```

Comment:
can someone weigh in on whether this is in the right place or needs a v32Schema? My assumption is that we only need 1 new schema per app release, but I'm not actually sure about that

Reply:
I am also not sure about when upgrades are needed. May need to ask the retention squad